fixtures/office_structure/update_node_fixtures.py

```diff
@@ -0,0 +1,68 @@
+update_node_mutation = '''
+mutation {
+  updateNode(nodeId: "C56A4180-65AA-42EC-A945-5FD21DEC0518",
+             name: "Epic Tower", tag: "Building Name") {
+    node {
+      name
+      tag
+    }
+  }
+}
+'''
+
+update_node_trim_spaces_mutation = '''
```

Comment:
@EmmanuelSage I don't think this test is necessary as it does not even improve coverage. You can instead, test for when `name` is supplied and `tag` is not supplied and vice-versa to improve the coverage.

Reply:
Thank you for the review, I added the test because it was a functionality of the method, just in case of refactoring in the future, but I could remove it thank you.
I already covered the lines for name and tag, as I only passed one or the other in the tests for empty name and tag, but I could write an explicit test for that,
Thank you.